l10n_ch_import_cresus/tests/test_import.py

```diff
@@ -42,32 +47,63 @@ def setUp(self):
     def test_import(self):
         journal_obj = self.env['account.journal']
         misc = journal_obj.search(
-            [('name', 'ilike', 'miscellaneous')], limit=1)
+            [('code', '=', 'MISC')], limit=1)
+        if not misc:
+            misc = journal_obj.create({
+                'code': 'MISC',
+                'name': 'dummy MISC',
+                'type': 'general'})
+
+        def get_path(filename):
+            res = get_resource_path('l10n_ch_import_cresus', 'tests', filename)
+            return res
+        input = open(get_path('input.csv'))
```

Comment:
Please don't use built-in function name for your variable.

Let's say you remove this line later, you won't get an error of undefined variable.